tests/pytest/test_multi.py

```diff
@@ -616,6 +623,14 @@ def testObjKeysCommand(env):
     res = r.execute_command('JSON.OBJKEYS', 'doc1')
     r.assertEqual(res, ["nested1", "a", "nested2"])
 
+    r.assertOk(r.execute_command('JSON.SET', 'doc2', '$', '{"a":{}, "nested":{"a":{"buz":"zer"}}}'))
+    res = r.execute_command('JSON.OBJKEYS', 'doc2', '$..a')
+    r.assertEqual(res, [[], ["buz"]])
+
+    r.assertOk(r.execute_command('JSON.SET', 'doc2', '$', '{}'))
+    res = r.execute_command('JSON.OBJKEYS', 'doc2', '$')
+    r.assertEqual(res, [[]])
+
```

Comment:
How does those tests related to this PR?

Reply:
Right, unrelated.